src/d3a/d3a_core/util.py

```diff
@@ -527,6 +537,25 @@ def convert_area_throughput_kVA_to_kWh(transfer_capacity_kWA, slot_length):
         if transfer_capacity_kWA is not None else 0.
 
 
+def find_timestamp_of_same_weekday_and_time(indict, time_slot, ignore_not_found=False):
```

Comment:
Again, what do you mean by `ignore_not_found `?

Reply:
This parameter is only used in the `read_user_profile._fill_gaps_in_profile` Where the gaps in the profiles are filled with the last available non-0 value. There, it is ok to not find the timestamp in the input profile.